rust-toolchain

```diff
@@ -0,0 +1 @@
+1.82.0
```

Comment:
```suggestion
1.83.0
```

Reply:
Nevermind, we have to update pyo3 for this to be compatible... It'll have to be another PR